properties/P000129.md

```diff
@@ -1,9 +1,9 @@
 ---
-uid: P100052
+uid: P000129
 slug: trivial
-name: Trivial
+name: Indiscrete
 aliases:
-  - Indiscrete
+  - Trivial
```

Comment:
From https://en.wikipedia.org/wiki/Trivial_topology: "a topological space with the _trivial topology_ is one where the only open sets are the empty set and the entire space. Such _spaces_ are commonly called _indiscrete_".
Also from Willard (3.2 b): "... called the trivial (indiscrete) topology ...".  Note that they never use the terminology "trivial space", only "trivial topology", although a space could be called "indiscrete space".

Based on that and what we discussed previously, would it be possible to replace "Trivial" here with "Trivial topology" as an alias?

And rephrase line 11 slightly as "A space $X$ is indiscrete or has the trivial topology if ..." maybe?